lib/remap-lang/src/lib.rs

```diff
@@ -281,6 +281,12 @@ mod tests {
                     "g": r#"{"1": Integer(1), "true": Boolean(true)}"#,
                 })),
             ),
+            ("fallible_func!()", Ok(()), Err("remap error: function call error: failed!")),
```

Comment:
I presume in time we will get a `fallible_func()` test?

Reply:
Good call. Let me add it in this PR, such that it'll fail once it becomes a compile-time instead of runtime error 👍